lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+class Menu
+  attr_accessor :menu
```

Comment:
would it be better to use attr_reader in here, so that the @menu cannot be altered somewhere else in the code?